groups/get_join_details.js

```diff
@@ -0,0 +1,145 @@
+const asyncAuto = require('async/auto');
+const asyncRetry = require('async/retry');
+const {connectPeer} = require('ln-sync');
+const {getChainBalance} = require('ln-service');
+const {getNodeAlias} = require('ln-sync');
+const {returnResult} = require('asyncjs-util');
+
+const {getGroupDetails} = require('./p2p');
+
+const coordinatorFromJoinCode = n => n.slice(0, 66);
+const groupIdFromJoinCode = n => n.slice(66);
+const interval = 500;
+const isCode = n => !!n && n.length === 98;
+const isPublicKey = n => !!n && /^0[2-3][0-9A-F]{64}$/i.test(n);
+const niceName = n => `${n.alias} ${n.id}`.trim();
+const times = 2 * 60;
+const tokensAsBigUnit = tokens => (tokens / 1e8).toFixed(8);
+
+/** Ask to confirm joining a group
+
+  {
+    ask: <Ask Function>
+    code: <Group Invite Code String>
+    lnd: <Authenticated LND API Object>
+    logger: <Winston Logger Object>
+    rate: <Max Fee Rate For Open Number>
```

Comment:
this variable doesn't seem to be used

Reply:
I logged all the details down below in one place. So pushed it along.

Reply:
the logging is actually logging the real fee, not the max fee though